libgit/browser.go

```diff
@@ -239,13 +260,18 @@ func (b *Browser) ReadDir(p string) (fis []os.FileInfo, err error) {
 		}
 	}
 
+	childrenPathsToCache := []string(nil)
```

Comment:
```go
childrenPathsToCache := make([]string, 0, len(dirTree.Entries))
```

Reply:
You know, sometimes when you write too much JavaScript, you tend to forget the performance sweetness like this from Go 😂